app/controllers/events/view/settings/options.js

```diff
@@ -67,6 +67,25 @@ export default class OptionsController extends Controller {
     try {
       this.set('isLoading', true);
       this.currentInvite.set('roleName', 'owner');
+      const invite =  this.model.roleInvites.filter(invite => invite.email === this.currentInvite.email)[0];
+      if (invite.id) {
+        const res = await this.loader.post('/role-invites/' + invite.id + '/resend-invite');
+        if (res.success) {
+          this.notify.success(this.l10n.t('Invite resent successfully'),
+            {
+              id: 'resend_invite_succ'
+            });
+        } else {
+          this.notify.error(this.l10n.t('Oops something went wrong. Please try again'));
+        }
+        this.setProperties({
+          'isConfirmEventTransferModalOpen' : false,
+          'checked'                         : false
+        });
+        this.set('isLoading', false);
```

Comment:
what need to set loading false here. it is already on line 100

Reply:
due to return statement that set loading will not work